Doc/library/pathlib.rst

```diff
@@ -1432,17 +1432,26 @@ Creating files and directories
 Copying, renaming and deleting
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. method:: Path.copy(target)
+.. method:: Path.copy(target, *, follow_symlinks=True)
 
    Copy the contents of this file to the *target* file. If *target* specifies
    a file that already exists, it will be replaced.
```

Comment:
Missed this in the earlier review, but what happens if `target` is a directory? (And more immediately relevant, what if `target` is a file symlink and `follow_symlinks=False`?)

Reply:
If `target` is an existing directory, you'll get an `OSError`, the exact flavour of which varies by OS. Same as the `shutil` copy functions.

If `target` is a symlink it will be following regardless of the value of `follow_symlinks`. Again this is the same as `shutil`. I've added a test for it.